components/EditCarForm/index.js

```diff
@@ -0,0 +1,68 @@
+const groups = [
+  {
+    id: 1,
+    headline: "Important data",
+    content: ["Make", "Model", "Milage", "Plate", "ImageUrl"],
+  },
+  {
+    id: 2,
+    headline: "Dimensions",
+    content: [
+      "Length (mm)",
+      "Height (mm)",
+      "Width (mm)",
+      "Width including mirrors (mm)",
+      "Height (mm)",
+      "Weight Empty (kg)",
+      "Max Weight (kg)",
+    ],
+  },
+  {
+    id: 3,
+    headline: "More Information",
+    content: [
+      "Drive",
+      "Model Year",
+      "Engine Displacement (ccm)",
+      "Transmission",
+      "Number Of Gears",
+    ],
+  },
+];
+
+export default function EditCarForm({ activeCar, onSubmit }) {
+  return (
+    <form onSubmit={onSubmit}>
+      {groups.map((group) => {
+        return (
+          <fieldset key={group.id}>
+            <p>{group.headline}</p>
+            {Object.keys(activeCar).map((attribute, index) => {
+              if (group.content.includes(attribute)) {
+                const attributeValue = activeCar[attribute];
```

Comment:
```suggestion
                const attributeValue = initialCars[attribute];
```
Here you need to address the json. else your input type="number" will change to an input type="text" as soon as you change it.